src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,35 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        //Excavator excavator = new Excavator();
+        //Bulldozer bulldozer = new Bulldozer();
+        //Truck truck = new Truck();
+
+        Object[] object = new Object[3];
+
+        object[0] = new Excavator();
+        object[1] = new Bulldozer();
+        object[2] = new Truck();
+
+
+        for (Object objects : object ) {
+            //Excavator
+            Excavator.doWork();
+            Excavator.stopWork();
```

Comment:
Similarly, 'stopWork' should be called on the instance of the class. Use the loop variable to call 'stopWork' on each specific machine instance.